assets/css/_vehicle_properties_panel.scss

```diff
@@ -66,6 +69,10 @@
   .on-time & {
     color: $color-component-medium;
   }
+
+  .off-course & {
+    color: $color-vehicle-early;
```

Comment:
I get that this is a red warning color, but it's not really "early". Should we add another color to the ui kit for using in warning cases like this, even if it's the same red as the early color, so we can be more semantic about it here?

(Same for below in `.m-vehicle-properties-panel__not-available {}`)